gluecodium/src/main/java/com/here/gluecodium/generator/dart/DartNameResolver.kt

```diff
@@ -100,7 +100,10 @@ internal class DartNameResolver(
                 val postfix: String
                 when (actualType) {
                     is LimeStruct -> {
-                        prefix = "${resolveName(actualType)}("
+                        val useDefaultsConstructor =
+                            actualType.fields.isNotEmpty() && limeValue.values.isEmpty()
+                        val constructorName = if (useDefaultsConstructor) ".withDefaults" else ""
```

Comment:
I think the name `.withDefaults` belongs in the name rules and ideally should be configurable.

Reply:
Created #120 for this.